packages/headless/src/features/facets/facet-search-set/facet-search-reducer-helpers.ts

```diff
@@ -54,6 +54,19 @@ export function handleFacetSearchUpdate<T extends FacetSearchResponse>(
   search.options = {...search.options, ...rest};
 }
 
+export function handleClearFacetSearchValues<T extends FacetSearchResponse>(
+  state: FacetSearchSetState<T>,
+  facetId: string
+) {
+  const search = state[facetId];
+
+  if (!search) {
+    return;
+  }
+
+  search.response.values = [];
```

Comment:
This is how the search box suggestions were implemented in Headless (there are `showSuggestions` and `hideSuggestions` in the Headless Search Box which just clear the values array), I decided to keep the behaviour consistent, but I generally agree that this is more of a UI problem than anything.

Reply:
Maybe this is something we can improve with the headless search box controller too? It might need to wait for 1.x though if it is breaking change.

In general, if an existing pattern leads to a slower experience, or unnecessary queries, I would prefer to avoid the pattern rather than continue down the path.

Reply:
I mean we could just mark the methods as deprecated inside headless and not use them in atomic.